packages/terra-date-input/src/DateInput.module.scss

```diff
@@ -64,29 +64,51 @@
     &.error {
       background-color: var(--terra-date-input-month-error-background-color, #fff);
       background-image: var(--terra-date-input-month-error-background-image);
+      border: var(--terra-date-input-month-error-border, 1px solid #e50000);
+      box-shadow: var(--terra-date-input-month-error-box-shadow, 0 0 0 1px #e50000);
```

Comment:
remove from here -- is added to the wrong location.

\
Go down to line 177-178 and update for `.month-select`:
```
&.error {
      border: var(--terra-date-input-month-select-error-border, 1px solid #e50000);
      box-shadow: var(--terra-date-input-month-select-error-box-shadow, 0 0 0 1px #e50000);
```

Reply:
Should be fixed here https://github.com/cerner/terra-framework/pull/923/commits/e1b103a65ed2a1fbc8cfcd4fb595a68c5307b269